pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -869,5 +869,6 @@ func cephVolumeEnvVar() []v1.EnvVar {
 	return []v1.EnvVar{
 		{Name: "CEPH_VOLUME_DEBUG", Value: "1"},
 		{Name: "CEPH_VOLUME_SKIP_RESTORECON", Value: "1"},
+		{Name: "DM_DISABLE_UDEV", Value: "1"},
```

Comment:
Can you add a comment here explaining the env var? I see it in the commit message, but would be good to have it next to the code as well.

Reply:
Some clarifying questions:
1. What is the benefit of doing it without udev versus with udev?
2. Am I understanding correctly that all ceph-volume based provisioning and OSDs will get this value?
3. Does this change also mean we no longer need to mount `/run/udev` (IIRC) in the container(s)?

Reply:
> Some clarifying questions:
> 
> 1. What is the benefit of doing it without udev versus with udev?

I guess the benefit is to bypass the natural difficulties we have to properly work with udev (exposing various sockets etc). I'm also hoping to remove potential races as well as speeding up LVM by only looking at /dev

> 2. Am I understanding correctly that all ceph-volume based provisioning and OSDs will get this value?

Correct, the filter is needed for both prepare and activate.

> 3. Does this change also mean we no longer need to mount `/run/udev` (IIRC) in the container(s)?

That's a good question, I haven't tried removing it, but I wouldn't mind removing it.
I guess I'm just afraid to break other scenarios (such as encryption).

Does that help?